spec/sec-class-definitions-patch.html

```diff
@@ -10,268 +10,398 @@ <h2>Syntax</h2>
       <ins>ClassStaticBlock</ins>
       `;`
 
-    <ins>
+    <ins class="block">
     ClassStaticBlock :
       `static` `{` ClassStaticBlockBody `}`
-    
+
     ClassStaticBlockBody :
       ClassStaticBlockStatementList
 
     ClassStaticBlockStatementList :
-      StatementList[~Yield, ~Await, ~Return]?
+      StatementList[~Yield, +Await, ~Return]?
     </ins>
   </emu-grammar>
 
   <emu-clause id="sec-class-definitions-static-semantics-early-errors">
     <h1>Static Semantics: Early Errors</h1>
-    <emu-grammar collapsed><ins>ClassStaticBlock: `static` `{` ClassStaticBlockBody `}`</ins></emu-grammar>
+    <ins class="block">
+    <emu-grammar>ClassStaticBlockBody: ClassStaticBlockStatementList</emu-grammar>
     <ul>
       <li>
-        <p><ins>It is a Syntax Error if HasDirectSuper of |ClassStaticBlock| is *true*.</ins></p>
+        <p>It is a Syntax Error if the LexicallyDeclaredNames of |ClassStaticBlockStatementList| contains any duplicate entries.</p>
       </li>
-    </ul>
-    <emu-grammar collapsed><ins>ClassStaticBlockBody: ClassStaticBlockStatementList</ins></emu-grammar>
-    <ul>
       <li>
-        <p><ins>It is a Syntax Error if the LexicallyDeclaredNames of |ClassStaticBlockStatementList| contains any duplicate entries.</ins></p>
+        <p>It is a Syntax Error if any element of the LexicallyDeclaredNames of |ClassStaticBlockStatementList| also occurs in the VarDeclaredNames of |ClassStaticBlockStatementList|.</p>
       </li>
       <li>
-        <p><ins>It is a Syntax Error if any element of the LexicallyDeclaredNames of |ClassStaticBlockStatementList| also occurs in the VarDeclaredNames of |ClassStaticBlockStatementList|.</ins></p>
+        <p>It is a Syntax Error if ContainsDuplicateLabels of |ClassStaticBlockStatementList| with argument &laquo; &raquo; is *true*.</p>
       </li>
       <li>
-        <p><ins>It is a Syntax Error if ContainsDuplicateLabels of |ClassStaticBlockStatementList| with argument &laquo; &raquo; is *true*.</ins></p>
+        <p>It is a Syntax Error if ContainsUndefinedBreakTarget of |ClassStaticBlockStatementList| with argument &laquo; &raquo; is *true*.</p>
       </li>
       <li>
-        <p><ins>It is a Syntax Error if ContainsUndefinedBreakTarget of |ClassStaticBlockStatementList| with argument &laquo; &raquo; is *true*.</ins></p>
+        <p>It is a Syntax Error if ContainsUndefinedContinueTarget of |ClassStaticBlockStatementList| with arguments &laquo; &raquo; and &laquo; &raquo; is *true*.</p>
```

Comment:
Sorry. Is this a typo?
![image](https://user-images.githubusercontent.com/6831019/121700078-1e150800-cb02-11eb-8d64-c853773899b1.png)

Reply:
Why would this be typo?

Reply:
Well.. Never mind.